src/region/transformUtils.ts

```diff
@@ -0,0 +1,24 @@
+export type Point = {
```

Comment:
I think we have this elsewhere, but not sure if it's in a common enough place to reuse.